domains/scratchempires.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "qloakongithub",
+           "email": "7469931481@stu.gcsnc.com",
+           "discord": "865563672553062431"
+        },
+    
+        "record": {
+            "CNAME": "https://scratch-empires-unofficial-organization.github.io/ScratchEmpiresSite/"
```

You'll have to go back to this record so that it will be valid, it will still be connectable though
```suggestion
            "CNAME": "scratch-empires-unofficial-organization.github.io"
```

---

Should of specified that I just needed the preview of the website

---

Okay, thanks.